pages/linux/hwinfo.md

```diff
@@ -0,0 +1,24 @@
+# hwinfo
+
+> Probe for the hardware present in the system.
+> More information: <https://github.com/openSUSE/hwinfo>.
```

Comment:
The man page is more informative.
```suggestion
> More information: <https://manned.org/hwinfo>.
```

Reply:
This is a bad change. The official repo is way more reliable than a third-party. Also according to rules of contributions of tldr, the official repo is a valid link